addons/point_of_sale/static/src/js/models.js

```diff
@@ -2268,6 +2268,14 @@ exports.Orderline = Backbone.Model.extend({
             return this.get_base_price();
         }
     },
+    get_taxed_lst_unit_price: function(){
+            var product =  this.get_product();
+            var taxes_ids = product.taxes_id;
+            var product_taxes = this.get_taxes_after_fp(taxes_ids);
+            var lst_price = this.get_lst_price();
+            var price = this.compute_all(product_taxes, lst_price, 1, this.pos.currency.rounding).total_included;
+            return price;
```

Comment:
* Reduce indentation.
* Can you also consider "iface_tax_included" since the result of this method is compared to "get_unit_display_price"?